README.md

```diff
@@ -127,17 +127,17 @@ Installing libffi
 
 First you must configure the distribution for your particular
 system. Go to the directory you wish to build libffi in and run the
-"configure" program found in the root directory of the libffi source
+`"configure"` program found in the root directory of the libffi source
 distribution.  Note that building libffi requires a C99 compatible
 compiler.
 
 If you're building libffi directly from git hosted sources, configure
-won't exist yet; run ./autogen.sh first.  This will require that you
-install autoconf, automake and libtool.
+won't exist yet; run `./autogen.sh` first.  This will require that you
+install `autoconf`, `automake` and `libtool`.
 
 You may want to tell configure where to install the libffi library and
 header files. To do that, use the ``--prefix`` configure switch.  Libffi
-will install under /usr/local by default.
+will install under `/usr/local` by default.
```

Comment:
I'm not a markdown expert, but I noticed that I used double backticks above for ``--prefix``.  What is the difference between single and double backticks?

Reply:
Not much difference. But i just thought it was most common to use single backticks in markdown